src/characters.rs

```diff
@@ -57,6 +59,45 @@ pub async fn get_characters(
             .collect();
         HttpResponse::Ok().json(characters_by_id)
     } else {
-        HttpResponse::BadRequest().json("no")
+        HttpResponse::InternalServerError().finish()
+    }
+}
+
+#[derive(Debug, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct NewAssignment {
+    actor_id: Option<i32>,
+    character_id: i32,
+}
+
+// Assign a character to an actor a boolean for whether the query went through
```

Comment:
Lol, I missed a word. Is this better? 
// Assign a character to an actor. Return a boolean for whether the query went through

Reply:
We're not actually returning a boolean, it's 200 and `true` for success and 500 for failure. No false is ever returned. Can we drop the `true` since it's not adding any extra information beyond the 200 success?

Reply:
The way the frontend is structured currently, it is expecting  some sort of json body. This is something we could rewrite in the future, but at the time I thought it was easier to return a boolean than to add another else if to handle non-json responses. Can we just return empty json? Or do you think it worthwhile to adjust the apiCall function?
` 
// api.js, apiCall()
if (response.ok) {
      return new ApiResponse(true, await response.json());
    } `

Reply:
🫤